site2/docs/io-quickstart.md

```diff
@@ -26,7 +26,9 @@ Before connecting Pulsar to a database, you need to install Pulsar and the desir
 
 Read [Run a standalone Pulsar cluster locally](getting-started-standalone.md) for downloading the Pulsar distribution.
 
-Pulsar releases a separate binary distribution to contain all the built-in connectors. To enable those connectors, you need to download the connectors tarball release:
+Pulsar releases a separate binary distribution to contain all the built-in connectors, see [here](https://archive.apache.org/dist/pulsar/pulsar-@pulsar:version@/connectors/).
```

Comment:
I'd suggest to jump to the [download page](https://pulsar.apache.org/download/) instead.

But there's no anchor yet, which I'm planning to convert the page into MDX and we will have ones.